_episodes/25.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: episode
+title:  "25: Trino going through changes"
+date: 2021-09-02
+tags: change data capture debezium kafka 
+youtube_id: "yuJ1r_xUcAo"
+wistia_id: "20kzmtvh31"
+sections: 
+   - title: "Concept of the week"
+     desc: "Change Data Capture"
+     time: 934
+   - title: "Concept of the week"
+     desc: "Debezium"
+     time: 1652
+   - title: "Concept of the week"
+     desc: "Debezium + Trino at Zomato"
+     time: 2747
+   - title: "PR of the week"
+     desc: "PR 4140 Implement aggregation pushdown in Pinot"
+     time: 3865
+   - title: "Question of the week"
+     desc: "Is there an array function that flattens a row into three rows?"
+     time: 4265
+---
+
+## Guests
+
+ * Ayush Chauhan, Data Platform Engineer at [Zomato](https://www.zomato.com/who-we-are)
+ ([Ayush Chauhan](https://www.linkedin.com/in/ayush-chauhan/)).
+ * Gunnar Morling, Lead of Debezium and Open source software engineer at [Red Hat](https://www.redhat.com)
+ ([@gunnarmorling](https://twitter.com/gunnarmorling)).
+ * Ashhar Hasan, Software Engineer at [Starburst](https://starburst.io/)
+ ([@hashhar](https://twitter.com/hashhar)).
+ 
+## Release 361
+
+Official announcement items from Martin:
+
+* Support for OAuth2/OIDC opaque access tokens
+* Aggregation pushdown for Pinot
+* Better performance for Parquet files with column indexes
+* Support for reading fields as JSON values in Elasticsearch
+
+Manfred's additional notes:
+
+ * Predicate pushdown in Cassandra
+ * Metadata cache size limitation in a few connectors
+ * Lots of improvements for Hive view support
+ * Glue table statistics improvements
+ 
+More info at <https://trino.io/docs/current/release/release-361.html>.
+
+## Concept of the week: Change Data Capture
+
+If you know Trino, you know it allows for flexible architectures that include 
+many systems with varying use cases they support. We've come to accept this 
+potpourri of systems as a general modus operandi for most businesses. 
```

Comment:
remove usage of potpurri and modus operandi to make more understandable internationally

Reply:
yeah together it makes this a bit wordy. I'll remove potpourri.